python/ray/experimental/sgd/sgd_worker.py

```diff
@@ -263,3 +264,28 @@ def ip(self):
 
     def warmup(self):
         warmup()
+
+    def get_params(self):
```

Comment:
https://github.com/ray-project/ray/blob/master/python/ray/experimental/sgd/mnist_example.py#L70 does the same thing